builtin/game/chat.lua

```diff
@@ -404,121 +385,111 @@ core.register_chatcommand("remove_player", {
 	end,
 })
 
+
+-- pos may be a non-integer position
+local function find_free_position_near(pos)
+	local tries = {
+		{x=1, y=0, z=0},
+		{x=-1, y=0, z=0},
+		{x=0, y=0, z=1},
+		{x=0, y=0, z=-1},
+	}
+	for _, d in ipairs(tries) do
+		local p = vector.add(pos, d)
+		local n = core.get_node_or_nil(p)
+		if n and n.name then
```

Comment:
`n.name` is guaranteed to exist when `n` is a table. Superfluous check.

Reply:
fixed